lib/contiguous.cpp

```diff
@@ -0,0 +1,52 @@
+#include "flag_gems/operators.h"
+#include "flag_gems/utils.h"
+
+#include <iostream>
+#include "c10/cuda/CUDAStream.h"
+#include "triton_jit/triton_jit_function.h"
+
+namespace flag_gems {
+using namespace triton_jit;
+
+at::Tensor contiguous(at::Tensor &input, at::MemoryFormat memory_format) {
```

Comment:
Use signature 
```c++
at::Tensor contiguous(const at::Tensor & self, at::MemoryFormat memory_format=c10::MemoryFormat::Contiguous);
```
since this is the signature translated from the schema

```yaml
contiguous(Tensor(a) self, *, MemoryFormat memory_format=contiguous_format) -> Tensor(a)
```

Reply:
Check this file torch/include/ATen/ops/contiguous_native.h